tests/ConfigTest.php

```diff
@@ -206,6 +209,25 @@ public function testRegisterCustomFixers(array $expected, iterable $suite): void
         self::assertSame($expected, $config->getCustomFixers());
     }
 
+    /**
+     * @param array<string, string> $ruleSets
+     *
+     * @dataProvider provideRegisterCustomRuleSetsCases
+     */
+    public function testRegisterCustomRuleSets(?string $expectedException, array $ruleSets): void
+    {
+        if (!empty($expectedException)) {
```

Comment:
```suggestion
        if (null !== $expectedException) {
```
https://localheinz.com/articles/2023/05/10/avoiding-empty-in-php/